csuntechlab/iSTART
# STR-56 -- FE: Add font-awesome

# Description
Added font-awesome; implemented font-awesome using  SVG via javascript.

# Testing
1. run `yarn install`
2. Visit any ``*.vue`` component.
3. Add **_`<i class="fas fa-prescription-bottle"></i>`_** into the template.
4. Visit the component you added the icon.
5. Ensure the icon is being rendered.
6. Check the DOM and confirm that the icon is being rendered as an SVG instead of a font-face.
